packages/components/src/Illustration/Spot/Spot.tsx

```diff
@@ -0,0 +1,191 @@
+import React from "react"
+import { Base, BaseProps } from "../subcomponents/Base"
+
+export type SpotProps = Pick<BaseProps, "alt" | "classNameOverride"> & {
+  enableAspectRatio?: boolean
+}
+
+const SPOT_ILLUSTRATION_BASE_PATH = "illustrations/heart/spot/"
+const createSpotIllustration =
+  (fileName: string) =>
+  // eslint-disable-next-line react/display-name
```

Comment:
I feel like there might be a better way here to give these display names instead of just JSX elements, but I decided not to fall down the refactor rabbit hole - best this could be looked at when we revisit the names of these